nsv13/code/modules/overmap/fighters/fighters.dm

```diff
@@ -65,6 +65,14 @@ After going through this checklist, you're ready to go!
 	var/warmup_cooldown = FALSE //So you cant blitz the fighter ignition in 2 seconds
 	var/ejecting = FALSE
 	var/throttle_lock = FALSE
+	var/has_escape_pod = /obj/structure/overmap/fighter/prebuilt/escapepod

```

Comment:
We will need to rewrite the smaller craft as a lot of it ends up being hacky code to keep it together. That is out of scope for this PR.
Escapepods cant be children of fighters